manifests/init.pp

```diff
@@ -113,8 +110,46 @@
   $wsgi_threads      = $sentry::params::wsgi_threads,
 ) inherits ::sentry::params {
 
+  class { '::sentry::setup':
```

Comment:
I think we can, now that I've made all the sub-classes private.  I'll try, and push a change if it's successful.